### ISSUE TITLE
Clipping geometry in makeSVG in nodejs

### ISSUE THREAD

**Issue by DraTeots** [root-project/jsroot#227]
Dear Sergey, 

I'm using makeSVG function from nodejs to make a picture of our geometry in a batch mode. The example that you provided for nodejs works out of the box. But I can't figure out how to pass draw options in particular clipping. According to [this documentation](https://github.com/root-project/jsroot/blob/master/docs/JSROOT.md#geometry-viewer) I tried clip[x,y,z], clip and clipxyz but since nothing worked I guess something different should go into `option` parameter or there is a bug. 

The minimal code: 

```js
let jsroot = require("jsroot");
let fs = require("fs");
jsroot.openFile("https://eicweb.phy.anl.gov/EIC/detectors/athena/-/jobs/559705/artifacts/raw/geo/calorimeters_geo.root?inline=false")
      .then(file => file.readObject("default"))
      .then(obj => jsroot.makeSVG( { object: obj, option: "clip", width: 1200, height: 800 }))
      .then(svg => { fs.writeFileSync("lego2.svg", svg); console.log(`Create lego2.svg size ${svg.length}`); });
```

Still the picture I get is

<img width="434" alt="image" src="https://user-images.githubusercontent.com/4732545/152209357-4607c6d1-d9ff-46c5-a30b-1aff1a147188.png">

**Comment by linev** [comment]
Hi,

Up to now clipping was disabled for the batch mode.
Mostly because plain SVG renderer was used before.

But since a while real `gl` renderer used in `node` and it can process clipping!

[This SVG](https://github.com/linev/jsroot-test/blob/master/TGeo/cms.svg) should be the same as [jsroot example](https://root.cern/js/latest/examples.htm#tgeo_cms). I just update it.

I will test more tomorrow and enable clipping in current master and also in previous releases.

**Comment by DraTeots** [comment]
Thank you so much! Especially for the promt response!

**Comment by linev** [comment]
I enable clipping in current `master` and in `6.3` branch.
If you need it in released version, I can tag `6.3.4` version and publish in npm.
